File: 21400654-BETANCOURT-ESPERICUETA-JESUS-ISMAEL/Ejercicios/ClasificarNum/ClasificarNumero.hs
clasificarNumero :: Int -> String
clasificarNumero n 
    | n < 0 = "Negativo"
    | n == 0 = "Cero"
    | otherwise = "Positivo"

main :: IO ()
main = do

    putStrLn "Clasificación de -5"
    putStrLn (clasificarNumero (-5))

    putStrLn "Clasificación de 0: "
    putStrLn (clasificarNumero 0)

    putStrLn "Clasificación de 10: "
    putStrLn (clasificarNumero 10)

File: 20400788-ORTEGA-DELUNA-HANNIA-ALI/ClasificarNumero.py
clasificarNumero :: Int -> String
clasificarNumero n 
 | n < 0     = "Negativo"
 | n == 0    = "Cero"
 | otherwise = "Positivo"

main :: IO ()
main = do

  putStrLn "Ingresa un número para clasificarlo:"
  input <- getLine
  let numero = read input :: Int
  putStrLn $ "El número es: " ++ clasificarNumero numero


 --putStrLn "Clasificación de -5:"
 --putStrLn (clasificarNumero (-5))

 --putStrLn "Clasificación de 0:"
 --putStrLn (clasificarNumero (0))

 --putStrLn "Clasificación de 10:"
 --putStrLn (clasificarNumero (10))